src/game/SpellEffects.cpp

```diff
@@ -1149,6 +1149,13 @@ void Spell::EffectDummy(SpellEffectIndex eff_idx)
 
                     return;
                 }
+                case 32027:                                 // Expedition Flare
+                {
+                    if (!m_CastItem)                        // 32029 = Expedition Preserver | 32030 = Expedition Scout
```

Comment:
Because you add it to CastSpell, which I usually avoid, at least on voa end, because when I am designing stuff I test it without items, unless it provides some special, needed functionality. Maybe the m_CastItem parameter in CastSpell isnt even needed in this instance.